src/components/atoms/SfAlert/SfAlert.html

```diff
@@ -0,0 +1,19 @@
+<div class="sf-alert" :class="`sf-alert--${type}`">
+    <slot>
+        <slot name="icon" :icon="icon" v-if="icon">
+            <div class="sf-alert__icon" >
+               <img :src="icon" v-if="typeof icon === 'string'">
```

Comment:
```suggestion
               <img :src="icon" v-if="icon'">
```

Reply:
It's because icon prop can be either string or boolen, and we want to display this icon only when prop is string(icon url) because default case it's true and it displays default icon.